app/src/components/task-completion/TaskUpload.tsx

```diff
@@ -67,15 +67,27 @@ type TaskUploadProps = {
   imageFileSrc: any;
   setImage: React.Dispatch<React.SetStateAction<any>>;
   setImageFileSrc: React.Dispatch<React.SetStateAction<any>>;
+  setInvalidImageMessage: React.Dispatch<React.SetStateAction<any>>;
 }
 
 export default function TaskUpload(props: TaskUploadProps) {
-  const { imageFileSrc, setImage, setImageFileSrc } = props;
+  const { imageFileSrc, setImage, setImageFileSrc, setInvalidImageMessage } = props;
 
   const onImageChange = (event: any) => {
+    console.log('event.target.files[0]',  event.target.files[0])
     if (event.target.files && event.target.files[0]) {
+      const allowedTypes = ['image/jpeg', 'image/jpg', 'image/png']
```

Comment:
might have to add `heic` for iphones?

Reply:
<img width="373" src="https://github.com/sendchinatownlove/apham-ecaao/assets/60532744/11fd9259-56ee-4a9b-b997-647431e6da37">

for HEIC photos, a preview is no longer shown. Is that ok? I did try uploading a file through my iPhone on an online code editor with <input> and it seems to convert live photos taken and camera roll images to jpeg/jpg/png

Reply:
Ah great! If it auto converts then no big deal, we can accept or not accept them, either way is fine by me!